image_analysis/timetable_analysis.py

```
@@ -1,522 +1,13 @@
-
-    merged = _merge_adjacent_same_name(schedule_rows)
+router = APIRouter()
 
-    # 메모리 확보
-    del base_gray
-    del vmask
-    del hmask
 
-    return merged
 @router.post("/timetable_analysis")
 async def upload_timetable(request: Request, file: UploadFile = File(...)):
```

Ruff (0.15.11)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)